coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -57,6 +57,9 @@ def parse(self):
         elif self.language == "python" and self.docstyle == "doxygen":
             return self._parse_documentation_with_symbols(
                 ("@param ", " "), "@return ")
+        elif self.language == "java" and self.docstyle == "default":
+            return self._parse_documentation_with_symbols(
+                ("@param  ", " "), "@return ")
```

Comment:
hey, don't we have coalangs from @Makman2 so we don't need if's for every language and style? Hardcoding stuff is bad :(